easybuild/easyconfigs/b/Bio-EUtilities/Bio-EUtilities-1.76-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,88 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Bundle'
+
+name = 'Bio-EUtilities'
+version = '1.76'
+
+homepage = 'https://github.com/bioperl/bio-eutilities'
+description = "BioPerl low-level API for retrieving and storing data from NCBI eUtils"
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+builddependencies = [('binutils', '2.32')]
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('BioPerl', '1.7.2'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+exts_list = [
+    ('ExtUtils::MakeMaker', '7.44', {
+        'source_tmpl': 'ExtUtils-MakeMaker-7.44.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BI/BINGOS'],
+        'checksums': ['52a18f8271250faf5f3527499dd2b78d3b4fd3b064408d0cfdda9a3538887188'],
+    }),
+    ('IO::Handle', '1.42', {
+        'source_tmpl': 'IO-1.42.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TO/TODDR'],
+        'checksums': ['eec5cc33a6cddba8b5d2425b60752882add7e4d41b7431a0ea4dcd73cc1f8cca'],
+    }),
+    ('Test::More', '0', {
+        'source_tmpl': 'Test-Simple-1.302171.tar.gz',
```

Comment:
`Test-Simple` or `Test-More`?

Reply:
ehm, good catch.
https://metacpan.org/pod/Test::More has download link to https://cpan.metacpan.org/authors/id/E/EX/EXODIST/Test-Simple-1.302175.tar.gz

Based on this: https://perlmaven.com/moving-over-to-test-more
>Let's start by the simple fact that Test::More lives in the same distribution that contains Test::Simple and that distribution is itself called Test-Simple.

Reply:
I've scraped the DL links from CPAN by script.

Reply:
Anyway this is included in Perl now.